src/instructlab/sdg/utils/taxonomy.py

```diff
@@ -109,6 +109,18 @@ def _get_taxonomy(repo="taxonomy"):
     return taxonomy_file_paths
 
 
+def _string_contains_html(s: str) -> bool:
+    """Detect HTML tags in a string.
+
+    We use this to catch markdown files that may contain html elements since
+    docling does not support this."""
```

Comment:
Is this a temporary fix until Docling supports html within markdown? Or is this something we expect to keep longer-term? We might want to open a feature request to docling for it to better handle this scenario so we can work towards removing this check entirely.

Reply:
https://github.com/DS4SD/docling/issues/734 Added a feature request for this.

Reply:
Thanks!